app/views/posts/show.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Single post page</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Post Details</title>
+</head>
+<body>
+<div class="post-card">
+      <p class="comt-likes"><strong>Comments:</strong> <%= @post.comments.count %> &nbsp;&nbsp; <strong>Likes:</strong> <%= @post.likes.count %></p>
+  <h5><%= @post.title %> by <%= @user.name %></h5>
+  <p><%= @post.text %></p>
+</div>
+<div class="post-cmts">
+  <h2>comments:</h2>
+  <% @post.comments.each do |comment| %>
+    <p><strong><%= comment.author.name %>:</strong> <%= comment.text %></p>
+  <% end %>
+</div>
```

Comment:
- [ ] Excellent job displaying the comments 👏. However, as seen in the wireframe, only the last 5 comments are shown. So, please do the same, but only show the last 5 comments.  :ok_hand: 😊

To display only the most recent posts, make sure to loop through 'return_five_most_recent_comments'.

```suggestion
 <% post.return_five_most_recent_comments.each do |comment| %>
        <p><strong><%= comment.author.name %>:</strong> <%= comment.text %></p>
  <% end %>
```